sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Microsoft.Azure.WebJobs.Extensions.EventHubs.csproj

```diff
@@ -3,9 +3,9 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <Description>Microsoft Azure WebJobs SDK EventHubs Extension</Description>
-    <Version>5.1.0-beta.1</Version>
+    <Version>5.1.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>5.0.1</ApiCompatVersion>
+    <ApiCompatVersion>5.1.0</ApiCompatVersion>
```

Comment:
This did not get updated because 5.1.0 was shipped out of a feature branch and not merged back to main.